### PR TITLE
misc: Updated README for binaries and crates.io

### PR DESCRIPTION
**Pull request by desertsagebrush**
Just some minor updates to the README now that packages are available on crates.io and built for each release. Hopefully, this will make it easier for folk to know where to find things and not turn people with issues against rust away.

I left the comment about automatic driver installation being experimental on windows, as I'm not sure if that has changed (though it has worked fine for me whenever I've tried it (not very often :sweat_smile:).

Glad to adjust anything if you want it changed.

Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,30 +8,38 @@ This tool can currently be used to update the Black Magic Debug firmware on your
 
 ## Installation
 
+Binary releases for Linux, Mac (x64/ARM) and Windows (x64/ARM) are now available with every
```
Comment (dragonmux) [inline README.md:11]: The proper name for the architectures are amd64 and AArch64, please use them in the ()'s (x64 is Microsoft's name for the architecture more formerly known as x86_64 or amd64, "ARM" refers normally to 32-bit ARM, aka AArch32)

File: README.md [hunk 2]
```diff
@@ -8,30 +8,38 @@ This tool can currently be used to update the Black Magic Debug firmware on your
...
+
 First install Rust on your computer. Follow the instructions on the [Rust Lang website](https://www.rust-lang.org/tools/install).
 
-Now you can build and install the tool from source.
+Then, install bmputil using `cargo install bmputil`
+
+bmputil on Windows will attempt to automatically setup driver installation on first run.
+This is extra experimental, and will require administrator access on the first run.
+
+## Building from source
```
Comment (dragonmux) [inline README.md:25]: Please insert a new line after this heading (it's a Markdown lint) and remove the extra one from line 15

File: README.md [hunk 3]
```diff
@@ -8,30 +8,38 @@ This tool can currently be used to update the Black Magic Debug firmware on your
...
-Driver Kit 8.0 redistributable components](https://go.microsoft.com/fwlink/p/?LinkID=253170) (link from
-[this](https://learn.microsoft.com/en-us/windows-hardware/drivers/other-wdk-downloads) page). If you are cross
-compiling to Windows, you will need to set the `WDK_DIR` environment variable to the path of the extracted WDK
-redistributable components.
+If you are working on patches or contributions to the tool, you can obviously use `cargo build` and
+`cargo run [params]` as needed.
 
-bmputil on Windows will also attempt to automatically setup driver installation on first run. This is extra
-experimental, and will require administrator access on the first run.
+### Windows
```
Comment (dragonmux) [inline README.md:37]: New line after the heading please

File: README.md [hunk 4]
```diff
@@ -8,30 +8,38 @@ This tool can currently be used to update the Black Magic Debug firmware on your
...
+For building bmputil locally for a windows platform (either on windows or cross-compiling), you will
```
Comment (dragonmux) [inline README.md:38]: windows -> Windows please

File: README.md [hunk 5]
```diff
@@ -8,30 +8,38 @@ This tool can currently be used to update the Black Magic Debug firmware on your
...
+need to install the [Windows Driver Kit 8.0 redistributable components](https://go.microsoft.com/fwlink/p/?LinkID=253170)
+(link from [this](https://learn.microsoft.com/en-us/windows-hardware/drivers/other-wdk-downloads) page).
+If you are cross compiling to Windows, you will need to set the `WDK_DIR` environment variable to
+the path of the extracted WDK redistributable components.
```
Comment (dragonmux) [inline README.md:43]: Our bad that there's an extra new line here, please clean it up with the other lint fixes.

**Review by dragonmux — CHANGES_REQUESTED** [review]
Thank you for the contribution, it's extremely appreciated! There are a couple of nomenclature/capitalisation items to address and a couple of Markdown lints, but with those sorted, we'll be very happy to merge this.

File: README.md (hunk 1 above)
Comment (desertsagebrush) [reply README.md:11]: This was just me being lazy; fixed.

**Comment by desertsagebrush** [comment]
Now that I remembered to add my changes before force-pushing, it should be good to go :)

**Review by dragonmux — APPROVED** [review]
LGTM, thank you! Merging.

**Pull request merged**